Examples/Example-iOS_Swift-Carthage/Source/AppAuthExampleViewController.swift

```diff
@@ -396,6 +397,7 @@ extension AppAuthExampleViewController {
             if let authState = authState {
                 self.setAuthState(authState)
                 self.logMessage("Got authorization tokens. Access token: \(authState.lastTokenResponse?.accessToken ?? "DEFAULT_TOKEN")")
+                self.logMessage("Got id_token token. id_token: \(authState.lastTokenResponse?.idToken ?? "DEFAULT_TOKEN")")
```

Comment:
Can we handle the nil case better? Seems factually incorrect to state that we got an id_token when we didn't. I don't see how printing "DEFAULT_TOKEN" helps anyone. 

Same comment could apply to the accessToken, although I would normally expect it to be nonnull.